Application/EdFi.Ods.AdminApi/Features/Applications/EditApplication.cs

```diff
@@ -58,66 +58,66 @@ private static void ValidateProfileIds(Request request, IUsersContext db)
-        [SwaggerSchema(Description = FeatureConstants.EducationOrganizationIdsDescription, Nullable = false)]
-        public IEnumerable<int>? EducationOrganizationIds { get; set; }
-
-        [SwaggerSchema(Description = FeatureConstants.OdsInstanceIdDescription, Nullable = false)]
-        public int OdsInstanceId { get; set; }
-    }
-
-    public class Validator : AbstractValidator<IEditApplicationModel>
-    {
-        public Validator()
-        {
-            RuleFor(m => m.Id).NotEmpty();
-            RuleFor(m => m.ApplicationName).NotEmpty();
-            RuleFor(m => m.ApplicationName)
-           .Must(BeWithinApplicationNameMaxLength)
-           .WithMessage(FeatureConstants.ApplicationNameLengthValidationMessage)
-           .When(x => x.ApplicationName != null);
-            RuleFor(m => m.ClaimSetName)
-                .NotEmpty()
-                .WithMessage(FeatureConstants.ClaimSetNameValidationMessage);
-
-            RuleFor(m => m.EducationOrganizationIds)
-                .NotEmpty()
-                .WithMessage(FeatureConstants.EdOrgIdsValidationMessage);
-
-            RuleFor(m => m.VendorId).Must(id => id > 0).WithMessage(FeatureConstants.VendorIdValidationMessage);
-            RuleFor(m => m.OdsInstanceId).Must(id => id > 0).WithMessage(FeatureConstants.OdsInstanceIdValidationMessage);
-
-            static bool BeWithinApplicationNameMaxLength<T>(IEditApplicationModel model, string? applicationName, ValidationContext<T> context)
-            {
-                var extraCharactersInName = applicationName!.Length - ValidationConstants.MaximumApplicationNameLength;
-                if (extraCharactersInName <= 0)
-                {
-                    return true;
-                }
-                context.MessageFormatter.AppendArgument("ApplicationName", applicationName);
-                context.MessageFormatter.AppendArgument("ExtraCharactersInName", extraCharactersInName);
-                return false;
-            }
-        }
-    }
-}
+    }
+
+    [SwaggerSchema(Title = "EditApplicationRequest")]
+    public class Request : IEditApplicationModel
+    {
+        [SwaggerExclude]
+        public int Id { get; set; }
+
+        [SwaggerSchema(Description = FeatureConstants.ApplicationNameDescription, Nullable = false)]
+        public string? ApplicationName { get; set; }
+
+        [SwaggerSchema(Description = FeatureConstants.VendorIdDescription, Nullable = false)]
+        public int VendorId { get; set; }
+
+        [SwaggerSchema(Description = FeatureConstants.ClaimSetNameDescription, Nullable = false)]
+        public string? ClaimSetName { get; set; }
+
+        [SwaggerOptional]
+        [SwaggerSchema(Description = FeatureConstants.ProfileIdDescription)]
+        public IEnumerable<int>? ProfileIds { get; set; }
+
+        [SwaggerSchema(Description = FeatureConstants.EducationOrganizationIdsDescription, Nullable = false)]
+        public IEnumerable<long>? EducationOrganizationIds { get; set; }

```

Comment:
This is the real change in this file. Data Standard 5.0 changed the `EducationOrganizationId` to a `long` instead of `int`. This fix is easy & opportunistic, although unrelated to the ticket.